### ISSUE TITLE
How to project face geometry vertices to match the face landmarks

### ISSUE THREAD

**Issue by apapaxiong** [cansik/mediapipe-extended#4]
Mediapipe has provided the landmarks coordinate. If I get the face geometry, geometry = results.multi_face_geometry[0], how can I project this geometry to match the landmarks (what camera intrinsic parameter to use?). It looks like the face geometry is around the origin (0,0,0).

**Comment by celyk** [comment]
I'm looking for a solution to this. The problem is that you can't access the camera parameters such as FOV, which are needed to construct the projection.

I think to access the parameters, `face_geometry.Environment` must be exposed ([source](https://github.com/google-ai-edge/mediapipe/blob/master/mediapipe/modules/face_geometry/README.md)).
> Describes an environment; includes the camera frame origin point location as well as virtual camera parameters.

I also found mention of `"//mediapipe/modules/face_geometry:env_generator_calculator"` in the [MediapPipe fork](https://github.com/cansik/mediapipe/commit/994806b090548e9b12259e275fb5894d5dd91f50)

**Comment by cansik** [comment]
@celyk Yes, this looks correct. I am not actively working on this project, but I am happy to review a PR.
